jvm/src/main/kotlin/app/cash/security_sdk/Certificate.kt

```diff
@@ -0,0 +1,65 @@
+package app.cash.security_sdk
+
+import app.cash.security_sdk.internal.providers.TrifleContentVerifierProvider
+import app.cash.security_sdk.internal.signers.TrifleContentSigner
+import okio.ByteString
+import org.bouncycastle.cert.X509CertificateHolder
+import org.bouncycastle.cert.X509v3CertificateBuilder
+import java.math.BigInteger
+import java.time.Instant
+import java.time.Period
+import java.util.Date
+
+/*
+ * Class representing a Trifle certificate.
+ *
+ * @property certificate - bytearray encoding of an x.509 certificate. Subject to change.
+ */
+class Certificate internal constructor(internal val certificate: ByteArray) {
+  /**
+   * Serialize the Trifle Certificate to a ByteArray so clients can store it for inclusion in future
+   * signing operations or provide it as a root certificate to verify signed messages.
+   */
+  fun serialize(): ByteArray {
+    return certificate
+  }
+
+  /**
+   * Signs CertificateRequest using this certificate as the issuer certificate, with its
+   * signing key (the TrifleContentSigner).
+   *
+   * @param certificateRequest certificate request used to generate a new certificate
+   * @param contentSigner content signer used to generate the signature validating the certificate
+   */
+  fun signCertificate(
```

Comment:
I dont think this is the right abstraction for where this method lives. If we were to take the `Certificate` class as a record type only object, it should only serve as either an input or output; we shouldnt need an instance of `Certificate` to possibly even issuing a different signing certificate.

The api we want to surface in this case `signCertificate` should remain in `CertUtil`.